tests/mysql-types-chrono.rs

```diff
@@ -63,6 +66,7 @@ async fn mysql_chrono_time() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[cfg(all(feature = "chrono", not(feature = "time")))]
```

Comment:
is that correct thing to do?
`chrono` and `time` features are not mutually exclusive in `sqlx-core`, but kinda in `sqlx-macros`

Reply:
This is only necessary for the one test using `query!()` since `time` supercedes `chrono` for expected input and output types. The dynamic `query()` function should still be able to work with both and thus shouldn't need this attribute. (Don't forget to fix this in `tests/postgres-types-chrono.rs` as well.)